python/ppml/example/fgboost_regression/fgboost-tutorial.md

```diff
@@ -89,12 +96,12 @@ Then call `fit` method to train
 Party 1:
 ```python
 # party 1 does not own label, so directly pass all the features
-fgboost_regression.fit(df_train, feature_columns=df_train.columns, num_round=100)
+fgboost_regression.fit(df_train, feature_columns=df_train.columns, num_round=10)
 ```
 Party 2:
 ```python
 # party 2 owns label, so pass the features and the labels
-fgboost_regression.fit(df_x, df_y, feature_columns=df_x.columns, label_columns=['SalePrice'], num_round=100)
+fgboost_regression.fit(df_x, df_y, feature_columns=df_x.columns, label_columns=['SalePrice'], num_round=10)
 ```
 ### 2.6 Save & Load
 Save the model and load it back
```

Comment:
- "Copy the configuration file `ppml/scripts/ppml-conf.yaml` to the directory where the server starts, and modify the config": what do you mean by "the directory where the server starts"?

- "Copy the configuration file ppml/scripts/ppml-conf.yaml to the directory where the client runs": what do you mean by "the directory where the client runs"?